<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogScharr</class>
 <widget class="QDialog" name="DialogScharr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,0">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>dy</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>dx</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>delta</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>bordertype</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QSlider" name="sliderDx">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSlider" name="sliderDy">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>scale</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSlider" name="sliderScale">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cBxBorderType">
     <item>
      <property name="text">
       <string>constant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>replicate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>reflect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>reflect_101</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="sliderDelta">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
